### PR TITLE
EPMRPP-89208 || Fix SonarCloud errors

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/pages/common/membersPage/membersPage.jsx [hunk 1]
```diff
@@ -122,7 +120,10 @@ export class MembersPage extends Component {
     }
     return (
       !!filter && (
-        <NoResultsForFilter filter={this.props.filter} notFoundMessage={messages.membersNotFound} />
+        <NoResultsForFilter
+          filter={this.props.filter}
+          notFoundMessage={this.props.intl.formatMessage(messages.membersNotFound)}
```
Comment (AmsterGet) [inline app/src/pages/common/membersPage/membersPage.jsx:125]: Why do you use `formatMessage` here? I see that this component format messages by itself.

File: app/src/pages/inside/common/searchableFilterList/searchableFilterList.jsx [hunk 2]
```diff
@@ -107,9 +107,13 @@ export class SearchableFilterList extends Component {
...
           noItemsMessage={
-            searchValue ? messages.filtersNotFound : messages.filtersNotFoundOnProject
+            searchValue
+              ? formatMessage(messages.filtersNotFound)
+              : formatMessage(messages.filtersNotFoundOnProject)
+          }
+          noItemsAdditionalMessage={
+            searchValue ? null : formatMessage(messages.filtersNotFoundAdditional)
           }
-          noItemsAdditionalMessage={searchValue ? null : messages.filtersNotFoundAdditional}
```
Comment (AmsterGet) [inline app/src/pages/inside/common/searchableFilterList/searchableFilterList.jsx:112]: The same for this file.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/common/searchableFilterList/searchableFilterList.jsx (hunk 2 above)
Comment (BlazarQSO) [reply app/src/pages/inside/common/searchableFilterList/searchableFilterList.jsx]: Done.

Comment (BlazarQSO) [reply app/src/pages/inside/common/searchableFilterList/searchableFilterList.jsx]: Done.

File: app/src/pages/common/membersPage/membersPage.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/membersPage.jsx]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**